content/welcome.md

```diff
@@ -16,3 +16,9 @@ __ＪＡＶＡＳＣＲＩＰＴ__.
 This workshop assumes you're running a recent-ish version of Node (>=4.0) and
 are comfortable with writing JS. Should you have any questions: there'll be
 mentor(s) to help out, so don't hesitate to ask. Good luck!
+
+- - -
+
+<sup>1</sup> This message is brought to you in part by [Irina
+Shestak](twitter.com/_lrlna) and [Yoshua Wuyts](twitter.com/yoshuawuyts). Terms
+and Conditions apply.
```

Comment:
Which terms and conditions? I don't know what this expression means; but afaik all guarantees we provide are covered by the license (if we have one).

Reply:
hahahahahah it's a joke. Like if you have an add they say `This message is brought to you in part by blah and blah. Terms and Conditions apply`. I'll remove the `terms and conditions` part; maybe not everyone will get it.